bin/src/app.rs

```diff
@@ -48,7 +48,9 @@ impl EthUIApp {
                 ethui_db::commands::db_get_contract_abi,
                 ethui_db::commands::db_get_erc20_metadata,
                 ethui_db::commands::db_get_erc20_balances,
+                ethui_db::commands::db_get_erc20_denylist,
                 ethui_db::commands::db_set_erc20_blacklist,
+                ethui_db::commands::db_set_erc20_allowlist,
```

Comment:
it's weird to call this allowlist because there actually isn't one. what this does is remove from the blacklist.

instead, I'd rather have `set_erc20_blacklist` and `clear_erc20_blacklist`

Reply:
Agree!